maintainer-guide.md

```diff
@@ -0,0 +1,79 @@
+# Maintainer's guide
+
+This guide is intended for maintainers — anybody with commit access to this repository.
+
+*Note:* This guide is a living standard;
+that is, it is meant to *describe* the project's maintenance practices,
+rather than *prescribe* them.
+As a maintainer, you're expected to refer to it for clarifications
+about the collaborative workflows of the project,
+but also to propose changes to it
+that you feel would make it more useful
+as a guideline for current and future maintainers.
+
+We use the [git flow methodology](http://nvie.com/posts/a-successful-git-branching-model/) for
+managing this repository. At a glance, this means:
+
+- a **master** branch. This branch MUST be releasable at all times. Commits and merges against
+  this branch MUST contain only bugfixes and/or security fixes. Maintenance releases are tagged
+  against master.
+- a **develop** branch. This branch contains *new features*, and will either become the next minor
+  (feature) release or next major release. Typically, major releases are reserved for backwards
+  *incompatible* changes, but can also be used to signal major new features.
+
+## I. Branch Naming Conventions
+
+- In addition to master and develop branches, these are the standards for features, fixes, chores and releases, 
+  1. **features** All feature branches must be created under **feat/**,
+  2. **bug-fixes** All fixes must be created under **fix/**,
+  3. **chores** Ad-hoc tasks that are not features, minor housekeeping, maintenance tasks should be created under **chores/**
+  4. Avoid branches being grouped under your usernames
+  
+## II. Handling PRs
+
+- When creating a PR, you should:
+  1. Clearly describe the intent of the PR 
+  2. Describe the solution in detail with links to the original issue and any related issues that it might fix or close. 
+  3. GitHub Draft PRs are a great way to get CI or human feedback on work that isn't yet ready to merge. PRs can be created as drafts and converted to normal PRs once the CI passes. More information about GitHub Draft PRs: https://github.blog/2019-02-14-introducing-draft-pull-requests/
+  
+
+- PRs should be merged once they
```

Comment:
Let's explicitly call this out here: Can anyone with access merge a PR? Some cultures have a preference for only the PR author to merge. I'm a fan of anyone with access merging. It's one fewer way to slow down the workflow.